pnpm-lock.yaml

```diff
@@ -7302,16 +7845,77 @@ packages:
       - encoding
     dev: true
 
-  /ember-source@5.9.0(@babel/core@7.24.5)(@glimmer/component@1.1.2)(@glint/template@1.4.0)(rsvp@4.8.5)(webpack@5.91.0):
+  /ember-source@5.9.0(@babel/core@7.24.7)(@glimmer/component@1.1.2)(@glint/template@1.4.0)(rsvp@4.8.5):
+    resolution: {integrity: sha512-sZdrRxsNJq49N+GlRpkrUfBagiCw5OUngXUcJO7tvoWPLTvO8RRip+1L2B868YkxlmSq22hLci9tgQUdmPmcXQ==}
+    engines: {node: '>= 16.*'}
+    peerDependencies:
+      '@glimmer/component': ^1.1.2
+    dependencies:
+      '@babel/helper-module-imports': 7.24.7(supports-color@8.1.1)
+      '@ember/edition-utils': 1.2.0
+      '@glimmer/compiler': 0.92.0
+      '@glimmer/component': 1.1.2(@babel/core@7.24.7)
+      '@glimmer/destroyable': 0.92.0
+      '@glimmer/env': 0.1.7
+      '@glimmer/global-context': 0.92.0
+      '@glimmer/interfaces': 0.92.0
+      '@glimmer/manager': 0.92.0
+      '@glimmer/node': 0.92.0
+      '@glimmer/opcode-compiler': 0.92.0
+      '@glimmer/owner': 0.92.0
+      '@glimmer/program': 0.92.0
+      '@glimmer/reference': 0.92.0
+      '@glimmer/runtime': 0.92.0
+      '@glimmer/syntax': 0.92.0
+      '@glimmer/util': 0.92.0
+      '@glimmer/validator': 0.92.0
+      '@glimmer/vm': 0.92.0
+      '@glimmer/vm-babel-plugins': 0.92.0(@babel/core@7.24.7)
+      '@simple-dom/interface': 1.4.0
+      babel-plugin-debug-macros: 0.3.4(@babel/core@7.24.7)
+      babel-plugin-ember-template-compilation: 2.2.5
+      babel-plugin-filter-imports: 4.0.0
+      backburner.js: 2.8.0
+      broccoli-concat: 4.2.5
+      broccoli-debug: 0.6.5
+      broccoli-file-creator: 2.1.1
+      broccoli-funnel: 3.0.8
+      broccoli-merge-trees: 4.2.0
+      chalk: 4.1.2
+      ember-auto-import: 2.7.3(@glint/template@1.4.0)
+      ember-cli-babel: 7.26.11
+      ember-cli-get-component-path-option: 1.0.0
+      ember-cli-is-package-missing: 1.0.0
+      ember-cli-normalize-entity-name: 1.0.0
+      ember-cli-path-utils: 1.0.0
+      ember-cli-string-utils: 1.1.0
+      ember-cli-typescript-blueprint-polyfill: 0.1.0
+      ember-cli-version-checker: 5.1.2
+      ember-router-generator: 2.0.0
+      inflection: 2.0.1
+      route-recognizer: 0.3.4
+      router_js: 8.0.5(route-recognizer@0.3.4)(rsvp@4.8.5)
+      semver: 7.6.2
+      silent-error: 1.1.1
+      simple-html-tokenizer: 0.5.11
+    transitivePeerDependencies:
+      - '@babel/core'
+      - '@glint/template'
+      - rsvp
+      - supports-color
+      - webpack
+    dev: false
+
+  /ember-source@5.9.0(@babel/core@7.24.7)(@glimmer/component@1.1.2)(@glint/template@1.4.0)(rsvp@4.8.5)(webpack@5.91.0):
```

Comment:
The second entry of `ember-source` (with a slightly difference set of dependencies, e.g. `webpack` is missing in the first entry) caused the following build error in `docs-app`:

```sh
docs-app@5.0.9 (dev)-> ember-container-query@5.0.9 -> ember-element-helper@0.8.6
    sees peerDep ember-source@5.9.0
      at /Users/isaaclee/Documents/GitHub/ember-container-query/node_modules/.pnpm/ember-source@5.9.0_@babel+core@7.24.7_@glimmer+component@1.1.2_@glint+template@1.4.0_rsvp@4.8.5/node_modules/ember-source
    but docs-app@5.0.9 is using ember-source@5.9.0
      at /Users/isaaclee/Documents/GitHub/ember-container-query/node_modules/.pnpm/ember-source@5.9.0_@babel+core@7.24.7_@glimmer+component@1.1.2_@glint+template@1.4.0_rsvp@4.8.5_webpack@5.91.0/node_modules/ember-source

docs-app@5.0.9 (dev)-> ember-container-query@5.0.9 -> ember-element-helper@0.8.6 -> @embroider/util@1.13.0
    sees peerDep ember-source@5.9.0
      at /Users/isaaclee/Documents/GitHub/ember-container-query/node_modules/.pnpm/ember-source@5.9.0_@babel+core@7.24.7_@glimmer+component@1.1.2_@glint+template@1.4.0_rsvp@4.8.5/node_modules/ember-source
    but docs-app@5.0.9 is using ember-source@5.9.0
      at /Users/isaaclee/Documents/GitHub/ember-container-query/node_modules/.pnpm/ember-source@5.9.0_@babel+core@7.24.7_@glimmer+component@1.1.2_@glint+template@1.4.0_rsvp@4.8.5_webpack@5.91.0/node_modules/ember-source
```